src/export/util.ts

```diff
@@ -0,0 +1,41 @@
+// (C) 2019 GoodData Corporation
+import { DashboardExport } from "@gooddata/typings";
+import { BAD_REQUEST_STATUS, SUCCESS_REQUEST_STATUS } from "../constants/errors";
+
+export function isDataExported(responseHeaders: Response): boolean {
+    const taskState = responseHeaders.status;
+    return taskState === SUCCESS_REQUEST_STATUS || taskState >= BAD_REQUEST_STATUS; // OK || ERROR
+}
+
+export function isDateFilter(
+    filter: DashboardExport.FilterContextItem,
+): filter is DashboardExport.IDateFilter {
```

Comment:
Why we are not reusing existing filter definitions which could be similar to what we are already using in AFM, VisualizationObject and Report export? it looks to me like inventing new wheel

Reply:
Yes, this type is neccessary to match backend spec.